src/Parser.js

```diff
@@ -73,7 +73,7 @@ export function parse (value, points, declarations, rules, rulesets) {
 							else if (attoken(atrule))
 								parse(temporary, points, children, [''], children)
 							else if (parse(value, points, props = [], rules, children), sizeof(props))
-								push(children, ruleset(value, points, 0, 0, rules, type, rules, props))
+								unshift(children, ruleset(value, points, 0, 0, rules, type, rules, props))
```

Comment:
`unshift` is, of course, worse in its perf than `push`, but i don't think there is anything better to do here as we only want to make this a first item in `children` conditionally and we know that it's going to be non-empty after parsing (and pushing to `children`) other stuff first

Reply:
It's faster than concat, so i say it's a win. This is only every done once for all the flat declarations in an at-rule nested rule block.

Reply:
Ok, this is ready to be merged then, right? 😃

Reply:
Ok, this is ready to be merged then, right? 😃

Reply:
Yes